storage/mysql/log_storage.go

```diff
@@ -325,6 +347,9 @@ func (t *logTreeTX) DequeueLeaves(ctx context.Context, limit int, cutoffTime tim
 	if rows.Err() != nil {
 		return nil, rows.Err()
 	}
+	label := labelForTX(t)
+	selectDuration := time.Now().Sub(start)
+	dequeueSelectLatency.Observe(float64(selectDuration/time.Millisecond), label)
```

Comment:
Would a local observeAsMillis(metric, time.Duration) func help to remove the boilerplate division?